packages/ag-charts-community/src/chart/series/seriesAreaManager.ts

```diff
@@ -735,18 +761,49 @@ export class SeriesAreaManager extends BaseManager {
                 this.chart.ctx.tooltipManager.removeTooltip(this.id);
             }
 
-            if (!refresh) {
-                // AG-13874 If all deltas are 0, it means that we're tabbing in (always announce). Otherwise, announce
-                // the datum pick only if the indices have changed.
-                const shouldAnnouncePick =
-                    (datumIndexDelta === 0 && otherIndexDelta === 0) ||
-                    oldDatumIndex !== pick.datumIndex ||
-                    oldOtherIndex !== (pick.otherIndex ?? focus.seriesIndex);
-                if (shouldAnnouncePick && tooltipContent != null) {
-                    this.swapChain.update(this.getDatumAriaText(datum, tooltipContent));
-                }
+            if (tooltipContent != null) {
+                this.maybeAnnouncePickedFocus(
+                    datumIndexDelta,
+                    oldDatumIndex,
+                    otherIndexDelta,
+                    oldOtherIndex,
+                    pick,
+                    tooltipContent
+                );
             }
         }
+
+        return PickedFocusStatus.SUCCESS;
+    }
+
+    private maybeAnnouncePickedFocus(
+        datumIndexDelta: number,
+        oldDatumIndex: number,
+        otherIndexDelta: number,
+        oldOtherIndex: number,
+        pick: PickFocusOutputs,
+        tooltipContent: TooltipContent[]
+    ) {
+        const { focus } = this;
+        let mode = this.announceMode;
+
+        if (mode === 'when-changed') {
+            // AG-13874 If all deltas are 0, it means that we're tabbing in (always announce). Otherwise, announce
+            // the datum pick only if the indices have changed.
+            const shouldAnnouncePick =
+                (datumIndexDelta === 0 && otherIndexDelta === 0) ||
+                oldDatumIndex !== pick.datumIndex ||
+                oldOtherIndex !== (pick.otherIndex ?? focus.seriesIndex);
+            if (shouldAnnouncePick) {
+                mode = 'always';
+            } else {
+                mode = 'never';
+            }
+        }
+
+        if ((mode satisfies 'always' | 'never') === 'always') {
```

Comment:
Nit: Is the `satisfies`/casting needed here? (Seems like it should be derivable automatically?)

Reply:
A more conventional way to encode this would be to create a variable `let mode: 'always' | 'never';` and assign to that in the `if` block - add a new `else` clause do `mode = this.announceMode` which will flag up the mismatch without harming readability.